packages/react-components/react-avatar/src/components/AvatarGroupOverflow/useAvatarGroupOverflowStyles.ts

```diff
@@ -8,15 +8,16 @@ import type { SlotClassNames } from '@fluentui/react-utilities';
 
 export const avatarGroupOverflowClassNames: SlotClassNames<AvatarGroupOverflowSlots> = {
   root: 'fui-AvatarGroupOverflow',
-  overflowButton: 'fui-AvatarGroupOverflow__overflowButton',
-  overflowContent: 'fui-AvatarGroupOverflow__overflowContent',
-  overflowSurface: 'fui-AvatarGroupOverflow__overflowSurface',
+  triggerButton: 'fui-AvatarGroupOverflow__triggerButton',
+  content: 'fui-AvatarGroupOverflow__content',
+  popoverSurface: 'fui-AvatarGroupOverflow__popoverSurface',
+  tooltip: 'fui-AvatarGroupOverflow__tooltip',
 };
```

Comment:
The tests won't allow it and the lint rules also will complain unless I add a comment to ignore it. While it doesn't make sense to have them, I do think that it feels worse not having them since I would need to both disable a test and add the comment.